tools/db_bench_tool.cc

```diff
@@ -1188,6 +1188,11 @@ DEFINE_bool(charge_file_metadata, false,
             "CacheEntryRoleOptions::charged of"
             "CacheEntryRole::kFileMetadata");
 
+DEFINE_bool(charge_blob_cache, false,
+            "Setting for "
+            "CacheEntryRoleOptions::charged of"
```

Comment:
Minor but a space seems to be missing after "of"

Reply:
Done